tests/epw_test.py

```diff
@@ -13,7 +13,7 @@ def test_get_universal_thermal_climate_index():
     calc_length = 8760
     relative_path = './tests/epw/chicago.epw'
     epw = EPW(relative_path)
```

Comment:
You can probably make the tests faster if you import epw once on top of the file and then reuse it between the methods.

Reply:
Ok. I'll do that. I guess I usually refrain from this practice in order to keep the tests as "true" unittests that can be separated from one another. But, because the purpose of that whole file is to test the workflows with EPWs, I think  your approach makes sense.

Reply:
Done.